docs/api-routes/api-middlewares.md

```diff
@@ -34,7 +34,9 @@ export const config = {
 
 The `api` object includes all configs available for API routes.
 
-`bodyParser` Enables body parsing, you can disable it if you want to consume it as a `Stream`:
+`bodyParser` Enables body parsing. If you want to consume it as a `Stream`, you can disable it.
```

Comment:
```suggestion
`bodyParser` allows disabling the automatic body parsing. If you want to consume the body as a `Stream` or with [`raw-body`](https://www.npmjs.com/package/raw-body), you can set this to `false`.
```
Should we mention this allows using the `raw-body` module manually as well?

Reply:
@ijjk, not setting it to `false` doesn't disable it. Is that accurate?

In that is accurate, I think explicitly mentioning is a good use case to convey the message.

 "`bodyParser` is automatically enabled. If you want to consume the body as a `Stream` or with [`raw-body`](https://www.npmjs.com/package/raw-body), you can set this to `false`."

Reply:
Yeah by default Next.js automatically consumes and parses the response body for API routes, if you want to consume it manually you need to set it to `false` to disable Next.js' built-in handling to do your own. Tweaked the suggestion a bit.

Reply:
Thanks for the explanation and the tweaking it 🙏